src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -8,7 +8,9 @@
 #include <sof/audio/module_adapter/module/iadk_modules.h>
```

Comment:
minor nit: cSOF is a term that has no meaning in SOF project, so please just use "SOF module".